Source/Core/DolphinQt/Debugger/WatchWidget.cpp

```diff
@@ -170,11 +170,11 @@ void WatchWidget::Update()
     {
       if (PowerPC::HostIsRAMAddress(entry.address))
       {
-        hex->setText(QStringLiteral("%1").arg(PowerPC::HostRead_U32(entry.address), 8, 16,
-                                              QLatin1Char('0')));
-        decimal->setText(QString::number(PowerPC::HostRead_U32(entry.address)));
+        hex->setText(QStringLiteral("%1").arg(PowerPC::HostRead_U32(entry.address).value_or(0), 8,
```

Comment:
Same down here, where `HostRead_U32` is called twice with the same parameters etc.